security/cmd/node_agent/server.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nodeagentmgmt
```

Comment:
for files other than main.go, the package name should generally match file dir, https://screenshot.googleplex.com/ebzTeCTHzcJ

Reply:
Fixed in the new commit.